internal/runbits/runtime/refresh.go

```diff
@@ -29,6 +40,16 @@ type Configurable interface {
 	GetBool(key string) bool
 }
 
+var overrideTriggers = map[target.Trigger]bool{
```

Comment:
```suggestion
var overrideAsyncTriggers = map[target.Trigger]bool{
```

Reply:
Makes it easier to understand what this is for without having to find how this var is used.